app/models/host_facets/reported_data_facet.rb

```diff
@@ -8,6 +8,7 @@ def self.populate_fields_from_facts(host, parser, type, source_proxy)
         ram: parser.ram,
         sockets: parser.sockets,
         cores: parser.cores,
+        disks_total: parser.disks_total,
```

Comment:
Shouldn't there be a `disks_total` in the [base FactParser](https://github.com/theforeman/foreman/blob/develop/app/services/fact_parser.rb)? Even if it returns nil, it'll at least prevent errors when a non-PuppetFactParser parser is used.